src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -508,4 +522,44 @@ void mongoSubCacheUpdate(const std::string& tenant, const std::string& subId, lo
       LM_E(("Internal Error (error updating 'lastNotification' for a subscription)"));
     }
   }
+
+
+  if (lastFailure > 0)
```

Comment:
I decided to use `> 0` as it is a bit safer. Captures both `0` and `-1`

Fixed in 73e190c